sdk/nodejs/runtime/state.ts

```diff
@@ -289,6 +332,13 @@ export const getStore = () => {
     return localStore;
 };
 
+export const getGlobalStore = () => {
```

Comment:
This should be marked as `@internal`. Also, it should go below the `(<any>getStore).captureReplacement = () => {` block, since that is modifying `getStore` it should be kept together.

Reply:
Done :heavy_check_mark: